packages/base-rollup-config/index.mjs

```diff
@@ -21,7 +21,9 @@
 import typescript from "@rollup/plugin-typescript";
 
 export const createSharedConfig = (pkg) => ({
-  plugins: [typescript()],
+  plugins: [typescript({
+    module: 'ESNext'
```

Comment:
In the example given in the docs, it looks like this should be wrapped in a `compilerOptions` object: https://github.com/rollup/plugins/tree/master/packages/typescript/#options. Do you have an example or a doc entry aligning with the way it is done here?

Reply:
The type definition allows for both; but lets go with the way the docs specify . Updated in https://github.com/inrupt/typescript-sdk-tools/pull/574/commits/d1541f17fd1f4898cf5fb82be3bd3d2ce817a164 and tested in https://github.com/inrupt/solid-client-js/pull/2290